docs/root/configuration/http_filters/grpc_json_transcoder_filter.rst

```diff
@@ -74,4 +81,81 @@ as its output message type. The implementation needs to set
 `content_type <https://github.com/googleapis/googleapis/blob/master/google/api/httpbody.proto#L68>`_
 (which sets the value of the HTTP response `Content-Type` header) and
 `data <https://github.com/googleapis/googleapis/blob/master/google/api/httpbody.proto#L71>`_
-(which sets the HTTP response body) accordingly.
\ No newline at end of file
+(which sets the HTTP response body) accordingly.
+
+
+Sample Envoy configuration
+--------------------------
+
+Here's a sample Envoy configuration that proxies to a gRPC server running on localhost:50051. Port 51051 proxies gRPC
+requests, while port 51052 uses the gRPC-JSON transcoder filter to provide the RESTful JSON mapping.
+
+.. code-block:: yaml
+
+  admin:
+    access_log_path: /tmp/admin_access.log
+    address:
+      socket_address: { address: 0.0.0.0, port_value: 9901 }
+
+  static_resources:
+    listeners:
+    - name: listener1
```

Comment:
I don't think you need this, listener2 can serve both gRPC and its RESTful JSON mapping.